util/missingcrypto.txt

```diff
@@ -672,6 +672,10 @@ IDEA_ofb64_encrypt(3)
 IDEA_options(3)
 IDEA_set_decrypt_key(3)
 IDEA_set_encrypt_key(3)
+OSSL_IETF_ATTR_SYNTAX_VALUE_free(3)
```

Comment:
This is a merge blocking change.
This file is the undocumented methods inventory.
It should not be extended.

There is a dedicated pod file to document these ASN1 framework-generated methods.

Reply:
man3/X509_dup.pod?  I'll move OSSL_IETF_ATTR_SYNTAX_VALUE_{free, it, new} there. (ACERT_INFO too)

What about the various d2i, i2d and _print functions? I've put those in this file because similar functions for the other X509 types are also undocumented.

Reply:
@FdaSilvaYY Thanks again for looking at this.

I've added documentation for most of the new API functions. The exceptions are the 
`X509_ACERT_print()` functions (which have a similar API to the `X509_print()` functions) and 
`X509_ACERT_add_attr_nconf()` (similar to `X509_ACERT_add_attr_nconf()`), which are currently undocumented.  

It's probably useful to add documentation for all of these versions (newly added and existing), but maybe in a separate PR?